main.cpp

```diff
@@ -52,85 +52,94 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun: Dog
+//  action 1: The dog barks.
+dog.bark();
+//  action 2: The dog sleeps.
+dog.sleep();
+//  action 3: The dog eats.
+dog.eat();
 
 //  2)
+//  Noun: Radio
+//  action 1: The radio receives signals.
+radio.receive();
+//  action 2: The radio transmits signals.
+radio.transmit();
+//  action 3: The radio changes frequencies.
+radio.setFrequency();

```

Comment:
These function names should match the plain English nearly identically.
```
radio.receiveSignal();
radio.transmitSignal();
radio.changeFrequency();
```
The purpose is so that your code reads exactly like the logic and thought patterns you had in your head when you wrote it. This way it's easy for you to understand what's happening and for other people to understand how you were approaching the problem that your code is solving. 
- [ ] revise